vars/perfRegressionParallelPipeline.groovy

```diff
@@ -81,6 +81,10 @@ def call(Map pipelineParams) {
                    description: 'Name of the test to run',
                    name: 'test_name')
 
+            string(defaultValue: "${pipelineParams.get('k8s_version', '')}",
```

Comment:
it's not enough. Need to also modify bash script in `Run ${sub_test}` section

Reply:
This pipeline calls `runSctTest` function which is updated as needed.
So, I don't see a problem. Update to the `runSctTest` function is the only one needed.